index.html

```diff
@@ -23,183 +23,216 @@
-                  <li>Codekit</li>
-                  <li>Codepen</li>
-                  <li>Github</li>
-                  <li>Gitlab</li>
-                  <li>Terminal</li>
-                 </ul>
-              </div>
-              <img id="teal-img" src="./images/image_dots_with_teal.png" alt="">
-            </div>
-          </div>
+    <div class="about-section-container">
+      <div class="about-section-left"><div id="about-section-list">
+        <div id="language">
+           <h5>Languages</h5>
+           <ul >
+                 <li>Javascricpt(ES5/ES6)</li>
+                 <li>Ruby</li>
+                 <li>HTML</li>
+                 <li>CSS</li>
+           </ul>
+ </div>
+ <div id="framework">
+      <h5>Frameworks</h5>
+     <ul>
+     <li>Bootsrap</li>
+     <li>Ruby on Rails</li>
+     <li>Rspec</li>
+     <li>Capibarra</li>
+     <li>Selenium</li> </ul>
+ </div>
+ <div id="skills">
+     <h5>Skills</h5>
+     <ul>
+     <li>Codekit</li>
+     <li>Codepen</li>
+     <li>Github</li>
+     <li>Gitlab</li>
+     <li>Terminal</li>
+    </ul>
+ </div>
+ <img id="teal-img" src="./images/image_dots_with_teal.png" alt="">
+</div></div>
+      <div class="about-section-right"><h2>About</h2>
+        <div id="about-sction-div">
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. 
+               Labore praesentium, ipsa necessitatibus maioresquaeiusto eveniet
+               asperiores at alias debitis deserunt excepturi, esse sequi.</p>
+             <button type="button">Get my cv   <img id="download" src="./images/arrow-right.png" alt=""></button>
+       </div>
+</div>
+    </div>     
+  </section> 
+  <section class="contact-form">   
+      <div  id="contact-form-img"> 
+        <img id="bg-contact-img1" src="./images/image-geometry6b.png" alt="">
+        <img id="bg-contact-img" src="./images/image-header.png" alt="">
       </div>
-  </section>    
-  <section class="contact-form">
-      <div id="bg-contact-img1"> <img src="./images/image3.png" alt=""></div>
-    <div id="bg-contact-img"></div>
-    <h2>Get Started</h2>
-    <div><p>Lorem ipsum, dolor sit amet consectetur adipisicing 
-      elit. Sint placeat dolorum perspiciatis rerum 
-      molestiae dicta aperiam libero quod officiis odio.</p></div>
-      <form action="https://formspree.io/f/mpzbbknz"  method="POST">
-          <div class="form-dir">
-        <label for="user_name" id="name"> <span>Name</span> 
-          <input class="input-field" type="text" id="user_name" placeholder="Enter your name"  required maxlength="30">
-        </label>
-
-        <label for="user_email">E-mail<input type="email" placeholder="Enter your email " id="user_email" required maxlength="30">
-        </label>
+      <div id="contact-form-txt"><h2>Get Started</h2>
+        <p>Lorem ipsum, dolor sit amet consectetur adipisicing 
+          elit. Sint placeat dolorum perspiciatis rerum 
+          molestiae dicta aperiam libero quod officiis odio.</p>
```

Comment:
[OPTIONAL]
- [ ] Also, kindly use the right font size as seen in the Figma design

<html>
<body>
<!--StartFragment-->

Figma design | Your design
-- | --
![image](https://user-images.githubusercontent.com/29071984/173187377-f2f40aa6-45f0-4398-8006-d3dd6680f27e.png) |  ![image](https://user-images.githubusercontent.com/29071984/173187364-76878396-1a08-47ba-bc85-f6a96fb9dc56.png)



<!--EndFragment-->
</body>
</html>